src/main/webapp/style.css

```diff
@@ -9,7 +9,7 @@
 
 img {
   max-width: 100%;
-  mad-heigh: 100%;
+  mad-height: 100%;
```

Comment:
hehe im sorry but it still says mad instead of max

Reply:
oh my goodness the typos will never end for me